interface/src/main.cpp

```diff
@@ -38,46 +42,46 @@ static BOOL CALLBACK enumWindowsCallback(HWND hWnd, LPARAM lParam) {
 #endif
 
 int main(int argc, const char* argv[]) {
-#ifdef Q_OS_WIN
-    // Run only one instance of Interface at a time.
-    HANDLE mutex = CreateMutex(NULL, FALSE, "High Fidelity Interface - " + qgetenv("USERNAME"));
-    DWORD result = GetLastError();
-    if (result == ERROR_ALREADY_EXISTS || result == ERROR_ACCESS_DENIED) {
-        // Interface is already running.
-        HWND otherInstance = NULL;
-        EnumWindows(enumWindowsCallback, (LPARAM)&otherInstance);
-        if (otherInstance) {
-            // Show other instance.
-            SendMessage(otherInstance, UWM_SHOW_APPLICATION, 0, 0);
-
-            // Send command line --url value to other instance.
-            if (argc >= 3) {
-                QStringList arguments;
-                for (int i = 0; i < argc; i += 1) {
-                    arguments << argv[i];
-                }
+    QString applicationName = "High Fidelity Interface";
```

Comment:
What happens if there's another user on the machine running interface? I'm guessing that's what the `qgetenv("USERNAME")` was for. 

We should remember to check that